src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -73,70 +66,74 @@ const ManageUserPage = ({
-                <FormFieldSelect
+      <LoadingSpinner loading={isLoading} text="Loading user">
+        <Formik<FormValues>
+          enableReinitialize
+          initialValues={{
+            userId: userId ?? '',
+            selectedRoleId: user?.role ?? '',
+          }}
+          validationSchema={Yup.object({
+            selectedRoleId: Yup.string().required('Choose role for the user'),
+          })}
+          onSubmit={handleSubmit}
+          render={_ => {
+            return (
+              <Form id={formId}>
+                <FormFieldset
+                  id={`${formId}-details`}
+                  legend="Detals"
+                  legendSize="m"
+                >
+                  <SummaryList>
+                    <SummaryListItem term="Name">{user?.name}</SummaryListItem>
+                    <SummaryListItem term="Email">
+                      {' '}
+                      {user?.email}
+                    </SummaryListItem>
+                    <SummaryListItem term="Phone">-</SummaryListItem>
+                  </SummaryList>
+                </FormFieldset>
+                <FormFieldset
                   id={`${formId}-role`}
-                  label="Role"
-                  name="userRole"
-                  // options={model.roles.map(role => ({
-                  //   label: role.name,
-                  //   value: role.id,
-                  // }))}
-                />
-              </FormFieldset>
+                  legend="Role"
+                  legendSize="m"
+                  hint="The users role within the service."
+                >
+                  <FormFieldSelect
+                    id={`${formId}-selectedRoleId`}
+                    label="Role"
+                    name="selectedRoleId"
+                    options={roles?.map(role => ({
+                      label: role.name,
+                      value: role.id,
+                    }))}
+                  />
+                </FormFieldset>
 
-              <FormFieldCheckboxGroup
+                {/* <FormFieldCheckboxGroup
                 id={`${formId}-releaseaccess`}
                 legend="Release access"
                 legendSize="m"
                 hint="The releases a user has access to (BAU Users have access to all releases)."
                 name="releaseAccess"
                 options={[]}
-              />
+              /> */}
 
-              <Button type="submit" className="govuk-!-margin-top-6">
-                Save
-              </Button>
-              <div className="govuk-!-margin-top-6">
-                <ButtonText onClick={cancelHandler}>Cancel</ButtonText>
-              </div>
-            </Form>
-          );
-        }}
-      />
+                <Button type="submit" className="govuk-!-margin-top-6">
+                  Save
+                </Button>
+                <div className="govuk-!-margin-top-6">
+                  <ButtonText onClick={cancelHandler}>Cancel</ButtonText>
```

Comment:
Not a big deal, but would  follow convention and call this `handleCancel`.

Reply:
Seems to be an even mix of both conventions